flink-end-to-end-tests/flink-end-to-end-tests-common/src/main/java/org/apache/flink/tests/util/flink/container/FlinkContainersBuilder.java

```diff
@@ -163,6 +163,8 @@ public FlinkContainers build() {
         this.conf.set(
                 CheckpointingOptions.CHECKPOINTS_DIRECTORY,
                 CHECKPOINT_PATH.toAbsolutePath().toUri().toString());
+        this.conf.set(RestOptions.ADDRESS, "0.0.0.0");
```

Comment:
This is basically an "advertised addressed". Since clients (individual tests) are using the exposed port [1] instead, we don't really have to set this.

[1] org.apache.flink.tests.util.flink.container.FlinkContainers#getJobManagerPort